app/Website/routes/website-routes.php

```diff
@@ -7,6 +7,7 @@
     Route::prefix('blog-posts')->group(function () {
         Route::get('/', [BlogPostController::class, 'index']);
         Route::get('/{slug}', [BlogPostController::class, 'show']);
+        Route::get('/tag/{tag}', [BlogPostController::class, 'listByTag']);
```

Comment:
This endpoint should probably be `/blog-post-tags/{tag}` defined outside of this route group, similar to /tags (which probably should be `/blog-post-tags` as well)